src/Employer/UnitTests/Employer.Web/Orchestrators/Vacancies/SearchResultHeaderTest/GivenSearchTermWithStatusFilter.cs

```diff
@@ -20,7 +20,7 @@ public async Task WhenThereAreNoVacancies()
         [Fact]
         public async Task WhenThereIsOneVacancy()
         {
-            var expectedMessage = "1 vacancy live with 'nurse'";
```

Comment:
Need to add tests for all the "live" options, also needs tests for non "live" options